functions/go/set-namespace/Dockerfile

```diff
@@ -1,8 +1,17 @@
 FROM golang:1.15
 ENV CGO_ENABLED=0
 WORKDIR /go/src/
-COPY . .
-RUN go build -v -o /usr/local/bin/function ./
+
+COPY go.mod go.mod
```

Comment:
Any benefit to explicitly copy files?

Reply:
In the first revision of this PR, I put the example directory along with the code instead of the top level examples directory.
Explicitly copy only needed files make it faster.
Given that the example has been moved to the top level directory, this change is not very effective now. I can revert this change.